### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Nadine2016**
# Description

Please include a summary of the changes.

This bug fixes ... \ This new feature can be used to ...

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/content/operations/upgrading/v1.14.md [hunk 1]
```diff
@@ -19,17 +19,27 @@ Review the following highlights of features changes in Gloo Edge version 1.14.
...
+* **Automatic discovery of GraphQL remote executors**: When your upstream service is already a GraphQL server that includes its own resolvers, you can automatically discover and use them to resolve requests for remote execution. You do not need to define another resolver within the GraphQLApi. For more information, see [Remote executor configuration for existing GraphQL server upstreams]({{% versioned_link_path fromRoot="/guides/graphql/resolver_config/#remote-executor-configuration-for-existing-graphql-server-upstreams" %}}). 
 
 **Deprecated features**:
 * AWS Lambda responseTransformation
+* [xDS relay]({{% versioned_link_path fromRoot="/operations/advanced/xds_relay/" %}})
```
Comment (sam-heilbron) [inline docs/content/operations/upgrading/v1.14.md:27]: I like this callout. I think we may even want a section of breaking changes. I see that in https://docs.solo.io/gloo-edge/latest/reference/changelog/open_source/#compareversions_v1.14.0-beta1...v1.14.0-rc2 we list a few. To be honest, I'm not sure how "breaking" they are, whether we truly stopped suporting something, or deprecated it. I'm happy to explore further if that helps

**Review by sam-heilbron — COMMENTED** [review]
These changes generally look good to me! A few scattered thoughts:

How in depth do we want the explanation of our new/improved features to be? Looking at the changelogs it seems we had listed 31 items as new features. I know we're not going to go in depth about each, just wasn't sure what balance we're trying to strike.

A major initiative from this quarter was to improve our scale testing to present some results. Perhaps that isn't going to be included in this 1.14 upgrade docs, but is there a plan (that you know of) to document those somewhere, akin to https://docs.solo.io/gloo-mesh-enterprise/latest/reference/scalability/

We now support kube 1.25. I believe this means that certain v1beta1 resources will no longer be supported. I'm not sure the best way to document that update, and am happy to strategize with you to find a way.

For xds-relay, I like the callout it is deprecated. Is there any reason we don't just remove the existing docs from being published?

**Comment by sam-heilbron** [comment]
test comment to see if bulldozer is applying the keep-pr-updated label

File: docs/content/operations/upgrading/v1.14.md (hunk 1 above)
Comment (Nadine2016) [reply docs/content/operations/upgrading/v1.14.md]: so I went through all the changes in the changelog and added Helm fields, new commands, and features to the upgrade notice. the xds doc will be removed as part of Art's PR so I removed the link as well.  I think in general, we want to call out major changes only as they can read about smaller features in the changelog themselves. hopefully I was able to do that with the update notice. :) 

Regarding performance numbers, I have it on my list to add docs for it.

**Review by sam-heilbron — DISMISSED** [review]

File: docs/content/operations/upgrading/v1.14.md [hunk 2]
```diff
@@ -55,7 +74,9 @@ New CRDs are automatically applied to your cluster when performing a `helm insta
 ## CLI changes
 
 **New CLI commands or options**:
-<!-- * None -->
+* New `--read-only` option in `glooctl check` to allow users to validate their configuration without requiring RBAC permissions on certain resources.
```
Comment (artberger) [inline docs/content/operations/upgrading/v1.14.md:77]: Maybe rephrase these to use 2nd person? or something to make the bullets parallel; right now they start with a noun phrase, past tense verb, and present tense verb.
```suggestion
* Validate your configuration without needing RBAC permissions on certain resources by using the `--read-only` option in `glooctl check`.
```

File: docs/content/operations/upgrading/v1.14.md [hunk 3]
```diff
@@ -55,7 +74,9 @@ New CRDs are automatically applied to your cluster when performing a `helm insta
 ## CLI changes
 
 **New CLI commands or options**:
-<!-- * None -->
+* New `--read-only` option in `glooctl check` to allow users to validate their configuration without requiring RBAC permissions on certain resources.  
+* Added the option to compare the CRDs that are currently installed with the CRDs of a specific Helm chart version by using the `glooctl check-crds --version <string> --local-chart <string> --show-yaml <bool>` command.
```
Comment (artberger) [inline docs/content/operations/upgrading/v1.14.md:78]:
```suggestion
* Compare the CRDs that are currently installed with the CRDs of a specific Helm chart version by using the `glooctl check-crds --version <string> --local-chart <string> --show-yaml <bool>` command. 
```

File: docs/content/operations/upgrading/v1.14.md [hunk 4]
```diff
@@ -55,7 +74,9 @@ New CRDs are automatically applied to your cluster when performing a `helm insta
 ## CLI changes
 
 **New CLI commands or options**:
-<!-- * None -->
+* New `--read-only` option in `glooctl check` to allow users to validate their configuration without requiring RBAC permissions on certain resources.  
+* Added the option to compare the CRDs that are currently installed with the CRDs of a specific Helm chart version by using the `glooctl check-crds --version <string> --local-chart <string> --show-yaml <bool>` command. 
+* Allow switching contexts for different clusters by using the `glooctl --kube-context <string>` command.
```
Comment (artberger) [inline docs/content/operations/upgrading/v1.14.md:79]:
```suggestion
* Switch contexts for different clusters by using the `glooctl --kube-context <string>` command. 
```

File: docs/content/operations/upgrading/v1.14.md [hunk 5]
```diff
@@ -19,23 +19,38 @@ Review the following highlights of features changes in Gloo Edge version 1.14.
...
+* **Added support for Kubernetes 1.25, removed support for 1.22**: Gloo Edge (OSS) can now be run on Kubernetes version 1.25. At the same time, support for Kubernetes version 1.22 was removed. This is considered a breaking change as APIs and file paths have changed. For example, the PodDisruptionBudget CRD moved from the `policy/v1beta1` to the `policy/v1` API. In addition, the Gloo Helm chart OCI registry path changed from `gcr.io/solo-public/gloo-helm` to `gcr.io/solo-public/gloo-helm/gloo`. 
+* **Horizontal pod autoscaling v2 support**: You can now configure horizontal pod autoscaling for the gateway proxy in accordance to the Kubernetes `autoscaling/v2` API. 
 
 **Deprecated features**:
 * AWS Lambda responseTransformation
 
 **Removed features**:
-<!-- * None -->
+* xDS relay
```
Comment (artberger) [inline docs/content/operations/upgrading/v1.14.md:31]: Not sure if we wanted a longer description, but if so, this is what I put in the 1.13:
```suggestion
* **xDS relay**: With the introduction of leader election for the Gloo Edge control plane, xDS relay is no longer needed to increase the availability of the xDS server. Previously, the xDS relay was deprecated in 1.13, a beta-only feature in Gloo Edge 1.11 or 1.12, and not available for non-default installation modes of Gloo Edge, such as REST Endpoint Discovery (EDS), mTLS, or Istio mTLS modes.
```

File: docs/content/operations/upgrading/v1.14.md [hunk 6]
```diff
@@ -19,23 +19,38 @@ Review the following highlights of features changes in Gloo Edge version 1.14.
...
 
 ## Helm changes {#helm}
 
 **New Helm fields**:
-<!-- * None -->
+* Added the ability to specify the security and pod security context for the redis and rate limit deployments by setting the following Helm values:
```
Comment (artberger) [inline docs/content/operations/upgrading/v1.14.md:36]: Make bullets parallel, either all past tense or all present tense; right now it's mixed. Maybe make 2nd person?
```suggestion
* Specify the security and pod security context for the Redis and rate limit deployments by setting the following Helm values:  
```

**Review by artberger — DISMISSED** [review]

**Review by artberger — DISMISSED** [review]

**Review by sam-heilbron — APPROVED** [review]

**Review by artberger — APPROVED** [review]
